CHANGES.md

```diff
@@ -1,18 +1,29 @@
 ringpop-go changes
 ==================
 
-v0.DEV (to be released)
------------------------
+v0.4.0
+------
 
-* Remove `File` and `Host`-based discover providers in favor of
-  `DiscoverProvider` interface #119
-* Ringpop will always assume the current host (`ringpop.node.identity`) is part
-  of the cluster. Previously, it was true for only `Host`-based discovery.
-* Add Go 1.6 testing on CI
+* Feature: Faulty nodes are now automatically reaped from nodes' membership
+  lists after 24 hours (by default). #123
+* New options for controlling suspect and reaping times. #123
+* Add new `Ready` and `Destroyed` events to ringpop. #125
+* Add additional logging to bring ringpop-go on par with ringpop-node log
+  messages. #116
+* Fix bug where Ringpop automatically added itself to the bootstrap hosts for
+  host-based bootstrapping, but not other bootstrapping methods. #120
+* Fix race condition where membership and hashring could be inconsistent with
+  each other. #112
+* Remove `File` and `Host` options from bootstrap options in favor of
+  `DiscoverProvider` interface. #120
+* Add Go 1.6 to testing on CI
```

Comment:
In the description, it's not 100% clear what is a changing behavior, versus internal change. I think it would be nice to split them out.

As an example, I would point to documentation of [Django release notes][1]: they are notorious on their backwards-compatibility promise, and in the way how they document it from release to release. Every described backwards-incompatible change is actionable.

[1]: https://docs.djangoproject.com/en/1.9/releases/1.9/#backwards-incompatible-changes-in-1-9